### PR TITLE
feat(teams): Team overview

### PR DESCRIPTION
**Pull request by hachiojidev**
image

### CODE REVIEW

File: src/views/Teams/Team.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+  useEffect(() => {
+    const fetchTeam = async () => {
+      try {
+        const pancakeProfileContract = getProfileContract()
+        const response = await pancakeProfileContract.methods.getTeamProfile(id).call()
+        const teamData = transformTeamResponse(response as TeamResponse)
+        setTeam(teamData)
+      } catch (error) {
+        setIs404(true)
```
Comment (hachiojidev) [inline src/views/Teams/Team.tsx:27]: Not the best solution...

File: src/views/Teams/components/TeamListCard.tsx [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+  team: Team
+}
+
+const getBackground = (theme: DefaultTheme) => {
+  if (theme.isDark) {
+    return 'linear-gradient(139.73deg, #142339 0%, #24243D 47.4%, #37273F 100%)'
+  }
+
+  return 'linear-gradient(139.73deg, #E6FDFF 0%, #EFF4F5 46.87%, #F3EFFF 100%)'
+}
```
Comment (hachiojidev) [inline src/views/Teams/components/TeamListCard.tsx:19]: Hopefully this is just temporary

File: src/views/Profile/ProfileCreation/TeamSelection.tsx [hunk 3]
```diff
@@ -73,25 +42,26 @@ const Team: React.FC = () => {
               'There’s currently no big difference between teams, and no benefit of joining one team over another for now. So pick whichever one you like!',
             )}
           </Text>
-          {teams.length === 0 && times(3).map((key) => <Skeleton key={key} height="80px" mb="16px" />)}
-          {teams.map((team, index) => {
-            // Team indices start at 1
-            const teamId = index + 1
+          {!teams && times(3).map((key) => <Skeleton key={key} height="80px" mb="16px" />)}
```
Comment (hachiojidev) [inline src/views/Profile/ProfileCreation/TeamSelection.tsx:45]: This is was the error we were talking about. Before it was an empty array, now the initial value `null`.

File: src/views/Teams/components/TeamCard.tsx [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+  padding-top: 0;
+  text-align: center;
+`
+
+const TeamName = styled(Heading).attrs({ as: 'h1' })`
+  font-size: 24px;
+
+  ${({ theme }) => theme.mediaQueries.md} {
+    font-size: 40px;
+  }
```
Comment (hachiojidev) [inline src/views/Teams/components/TeamCard.tsx:55]: These headers will be getting sorted out in the UI Kit.

File: src/views/Profile/ProfileCreation/TeamSelection.tsx (hunk 3 above)
Comment (RabbitDoge) [reply src/views/Profile/ProfileCreation/TeamSelection.tsx:45]: This 3 deserve its own variable , probably in a /config file

Comment (hachiojidev) [reply src/views/Profile/ProfileCreation/TeamSelection.tsx:45]: Ah yes, you mean the teams list.

File: src/views/Teams/Team.tsx (hunk 1 above)
Comment (RabbitDoge) [reply src/views/Teams/Team.tsx:27]: I can't believe react router doesn't allow for route param validation using a regex ....

**Review by RabbitDoge — APPROVED** [review]

**Pull request merged**